src/misc/lv_anim.h

```diff
@@ -146,6 +147,9 @@ struct _lv_anim_t {
 
     /* Animation system use these - user shouldn't set */
     uint32_t last_timer_run;
+    bool is_paused;                           /**<Indicates that the animation is paused */
```

Comment:
To save a few bytes please move it above `uint8_t reverse_play_in_progress : 1;`
```suggestion
    uint8_t is_paused :1;                           /**<Indicates that the animation is paused */
```